config/database.php

```diff
@@ -59,7 +59,7 @@
         'mysql' => [
             'host'      => env('DB_HOST', 'localhost'),
             'read' => [
-                'host' => env('DB_HOST_REPLICA', env('DB_HOST', 'localhost')),
+                'host' => !empty(env('DB_HOST_REPLICA')) ? env('DB_HOST_REPLICA') : env('DB_HOST', 'localhost'),
```

Comment:
so I have to explicitly check for empty sadly. This kinda sucks, it should be changed once we solve the underlying config problem we discovered.

Reply:
yep that's quite an eyeful 😄